src/medLog/medLogger.cpp

```diff
@@ -112,6 +110,9 @@ void medLogger::initializeTeeStreams()
 
 void medLogger::finalizeTeeStreams()
 {
+    QObject::disconnect(this, SIGNAL(newQtMessage(QtMsgType,QString)), this, SLOT(redirectQtMessage(QtMsgType,QString)));
+    qInstallMsgHandler(NULL);
```

Comment:
The problem is that `~medLogger()` is never called. Maybe we can change the name of the function if you want?